### PR TITLE
[MVS-478] Implemented date formatting within Patient Registration Component

### PR DESCRIPTION
**Pull request by TomMakes**
Changed formatting of date of birth string within the Patient Registration Component.

## :flipper: [JIRA Sprint Card]

https://mass-immunization-system.atlassian.net/browse/MVS-478

## :newspaper: [Work Description]

Changed formatting of date from 'Year-Month-Day' to 'Month/Day/Year', following a more normal date convention.

## :vertical_traffic_light: [Testing/QA Notes]

Run the patient registration application. Try to put in invalid dates and confirm the app throws errors when needed. Put in a valid date manually and through the date picker, proceed through registration workflow.
Check the console to see if errors are thrown in the process.

### CODE REVIEW

**Comment by ShahdSawalhi** [comment]
Does the scope of this card include the household work flow too? The single patient looks good though!

File: PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue [hunk 1]
```diff
@@ -174,26 +176,68 @@ export default {
...
+        // Ensure birthdate is fully entered and can be converted into 3 variables
+        if(birthdate) {
+          if(birthdate.split('/').length === 3) {
+            const [month, day, year] = birthdate.split('/');
+            return `${year}-${month.padStart(2, '0')}-${day.padStart(2, '0')}`;
+          }
+        }
+        return false;
+      }
+      var date = Date.parse(formattedDate());
```
Comment (david-fraley) [inline PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue:216]: I would prefer we keep variable names to be representative of their data and specific for what they represent, so maybe `dateOfBirth` here or something?

**Review by david-fraley — CHANGES_REQUESTED** [review]
Is the below expected UX? When I click on the DOB selector, I get this little shadow. Did you disable something? 

image

**Review by david-fraley — APPROVED** [review]

File: PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue (hunk 1 above)
Comment (TomMakes) [reply PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue:216]: I'm really not sure what the source of the date picker not coming up is. 
I had that happen once while I was developing, then never again after a refresh.

I'll try to see if I can reproduce.

Comment (TomMakes) [reply PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue:216]: I mainly called it date as naming the variable DOB was causing issues for me that I couldn't figure out quickly. I can dig around some more later.

**Pull request merged**